swiper.el

```diff
@@ -1240,11 +1240,11 @@ otherwise continue prompting for buffers."
       (t (buffer-file-name buffer)))))
 
 ;;* `swiper-all'
-(defun swiper-all-function (str)
-  "Search in all open buffers for STR."
+(defun swiper-all-buffers-function (str buffers)
```

Comment:
Do we want this function to be "public", i.e. without a double hyphen `--` in its name?

Reply:
Given that `swiper-all-function` is public, I felt like it was better to keep this one public too. 
Keeping it public signals that you could use this function to write your own search functions using this. Making that easier is one of the goals of this PR

Reply:
Fair enough.